code/cansat_code/wind/wind_reader.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python
+
+from wind import *
+from datetime import datetime
+import time
+file_datetable = open('/home/pi/data/wind/datetable.txt', 'a')
```

Comment:
```/home/pi/data/wind/``` - ten kawałek ścieżki do jakiegoś configa, albo chociaż do zmiennej, do not repeat yourself